src/RestfulServer.php

```diff
@@ -905,11 +905,11 @@ protected function getMember()
      * @param HTTPRequest $request
      * @return string
      */
-    protected function resolveClassName(HTTPRequest $request): string
+    protected function resolveClassName(HTTPRequest $request)
     {
         $className = $request->param('ClassName');
         $aliases = self::config()->get('endpoint_aliases');
 
-        return $aliases[$className] ?? $this->unsanitiseClassName($className);
+        return empty($aliases[$className]) ? $this->unsanitiseClassName($className) : $aliases[$className];
```

Comment:
I think you've got your operator back to front. Shouldn't this be `return !empty() ? $this->unsan... : $aliases..` ?

Reply:
Oh I see, sorry, it's early =D